js/app/dashboard/redux/actions.js

```diff
@@ -458,6 +458,16 @@ function closeImportModal() {
   return { type: CLOSE_IMPORT_MODAL }
 }
 
+function updateTask(task) {
+  return function(dispatch, getState) {
+    let date = selectSelectedDate(getState())
+
+    if (isInDateRange(task, date)) {
```

Comment:
This means that if the user changes the task date, changes are ignored. 

(see 7ee119308b301a172cdf9fb280bf8cd0240bb99e)